src/components/Lend/LoanSearch/LoanSearchFilterChips.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="tw-flex tw-flex-wrap tw-gap-1.5 tw-my-.5 tw-mb-1">
+		<div v-for="(item, i) in items" :key="i" class="tw-inline-block">
+			<kv-chip-classic :value="item.value">
```

Comment:
You won't need to pass a prop down here but you'll need to listen for `@click` and call a new method (to be defined below) with your full "item" object. This new method "handleChipClick" will receive your "item" and can then emit the update event and include a tracking event to represent the interaction.